elasticsearch/build.gradle.kts

```diff
@@ -37,6 +37,9 @@ dependencies {
 
     testImplementation("org.apache.logging.log4j:log4j-api")
     testImplementation("org.apache.logging.log4j:log4j-core")
+    // Elasticsearch doesn't export painless script artifact to maven central.
+    // Using 3rd party version (codelibs) only for test.
```

Comment:
Please use `because("...")` instead of comments

Reply:
@vlsi  I've made some changes if I understand you correctly.

Reply:
@ILuffZhe  You have some misunderstanding about the @vlsi comment. You can find an example in [https://github.com/gradle/gradle-profiler/blob/master/build.gradle.kts#L33](https://github.com/gradle/gradle-profiler/blob/master/build.gradle.kts#L33)

Reply:
Thanks for your reminder, @NobiGo. I've corrected it.